frontend/packages/webterminal-plugin/integration-tests/support/step-definitions/common/webTerminal.ts

```diff
@@ -10,29 +10,29 @@ import { checkTerminalIcon } from '@console/dev-console/integration-tests/suppor
 import { webTerminalPage } from '@console/webterminal-plugin/integration-tests/support/step-definitions/pages/web-terminal/webTerminal-page';
 
 const idp = Cypress.env('BRIDGE_HTPASSWD_IDP') || 'consoledeveloper';
-const username = Cypress.env('BRIDGE_HTPASSWD_USERNAME') || 'consoledeveloper';
+const username = Cypress.env('BRIDGE_HTPASSWD_BASIC_USERNAME') || 'consoledeveloper';
 const password = Cypress.env('BRIDGE_HTPASSWD_PASSWORD') || 'developer';
 const kubeAdmUserName = Cypress.env('KUBEADMIN_NAME') || 'kubeadmin';
 const kubeAdmUserPass = Cypress.env('BRIDGE_KUBEADMIN_PASSWORD');
 
 // create web terminal instance in dedicated namespace  under basic user and relogin as admin with oc client
-function installDevWsAndReconfigureIdlingTimeout(dedicatedNamespace: string) {
+function installDevWsAndWebTerminalForBasicUser(dedicatedNamespace: string) {
   // we need to create an active terminal session in background before test, also
   // we rewrite default idling timeout for the terminal (because 15 min - too long change it to 1 min)
   try {
     cy.exec(`oc login -u ${username}  -p ${password} --insecure-skip-tls-verify`);
-    cy.exec(`oc new-project  ${dedicatedNamespace}`);
-    cy.exec(`oc apply -f testData/yamls/web-terminal/web-terminal.yaml -n ${dedicatedNamespace}`);
+    cy.exec(`oc new-project  ${dedicatedNamespace} || oc project ${dedicatedNamespace}`);
+    // average time for project reconciling for slow infrastructure
+    cy.wait(5000);
```

Comment:
average waiting times has always two risks: In most cases, we wait more time than needed, and sometimes it still crashes.

Is it possible to use [oc wait](https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/developer-cli-commands.html#oc-wait) here instead?

Reply:
Replace to wait condition